documentation/source/users/arkane/input.rst

```diff
@@ -421,6 +421,44 @@ for specifying the same ``rotors`` entry are commented out)::
 
 Note that the atom labels identified within the rotor section should correspond to the indicated geometry.
 
+Multidimensional Hindered Rotors
```

Comment:
Could you please add general guidelines for users of which rotor type is recommended to use for different cases (stating that better accuracy means more computational time)? It would be helpful to add some species as an example, perhaps as a new section in the docs?

Reply:
The reality is that I've primarily run good reality checks rather than any good comparisons.  We don't have an easy way to generate the data required.  The only solid comparison I did was between semiclassical ND rotor, QM HIndered Rotor and  semiclassical HinderedRotor for Toluene.  In general those three are pretty much the same.  But I don't think that's enough to write guidelines.

Reply:
What species was used for the comparison? Can we tell users that ROOH's, for example, will probably required ND treatment?